api/v1/qdrantclustersnapshot_types.go

```diff
@@ -46,6 +46,9 @@ type QdrantClusterSnapshotStatus struct {
 	// The calculated time (in UTC) this snapshot will be deleted, if so.
 	// +optional
 	RetainUntil *metav1.Time `json:"retainUntil,omitempty"`
+	// Duration specifies how long it took for the snapshot to complete
+	// +optional
```

Comment:
why is this optional?

Reply:
This will be set only when backup is completed. If backup is running, this field will not exist. If we don't mark it as optional, we can't update the status for a running snapshot.

Reply:
Thanks for the clarification.